sections/header.liquid

```diff
@@ -11,6 +11,11 @@
   <link rel="stylesheet" href="{{ 'component-mega-menu.css' | asset_url }}" media="print" onload="this.media='all'">
   <noscript>{{ 'component-mega-menu.css' | asset_url | stylesheet_tag }}</noscript>
 {%- endif -%}
+{%- if section.settings.enable_country_selector or section.settings.enable_language_selector -%}
+  {{ 'component-localization-form.css' | asset_url | stylesheet_tag: preload: true }}
+  <noscript>{{ 'component-localization-form.css' | asset_url | stylesheet_tag }}</noscript>
```

Comment:
Exactly:
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/31-55-pariv-m8e87.png)
</details>

In some other places we rely on the `onload` attribute which requires JS. But here since we don't, we don't need a `no-script` I believe.